feature-libs/asm/customer-360/components/customer-360/avatar-image.pipe.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+import { CustomerOverview } from '@spartacus/asm/customer-360/root';
+import { Image } from '@spartacus/core';
+
+@Pipe({ name: 'avatarImage' })
```

Comment:
Nitpick, but almost all of pipes names in the storefront starts from "cx".  Maybe for consistency we should add "cx" prefix also.